docs/releasenotes/9.0.0.rst

```diff
@@ -72,6 +72,22 @@ Support has been added for the "title" argument in
 argument will also now be supported, e.g. ``im.show(title="My Image")`` and
 ``ImageShow.show(im, title="My Image")``.
 
+Added support for pickling TrueType fonts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+TrueType fonts may now be pickled and unpickled. For example:
+
+.. code-block:: python
+
+    import pickle
+    from PIL import ImageFont
+
+    font = ImageFont.truetype("arial.ttf", size=30)
+    pickled_font = pickle.dumps(font, protocol=pickle.HIGHEST_PROTOCOL)
+
+    # Later...
+    unpickled_font = pickle.loads(pickled_font)
+
```

Comment:
Feel free to disagree - but does this really belong under 'API Additions'?

Reply:
Good question, does it fit better under another heading? Or just leave it out of the release notes?

Reply:
Oh, I don't mind leaving it in the release notes, I just thought that "Other Changes" might be the more appropriate place.

Reply:
Sounds good, moved!